### PR TITLE
Fixing random JSDoc issues

### PR DESCRIPTION
**Pull request by kungfooman**
Main reason is they are causing type error spam every frame in Examples browser or other issues in RTI.js, e.g. the broken Vec4 import.

I confirm I have read the [contributing guidelines](https://github.com/playcanvas/engine/blob/master/.github/CONTRIBUTING.md) and signed the [Contributor License Agreement](https://docs.google.com/a/playcanvas.com/forms/d/1Ih69zQfJG-QDLIEpHr6CsaAs6fPORNOVnMv5nuo0cjk/viewform).

### CODE REVIEW

File: src/scene/skin-instance.js [hunk 1]
```diff
@@ -86,8 +86,14 @@ class SkinInstance {
...
 
-    // resolved skin bones to a hierarchy with the rootBone at its root.
-    // entity parameter specifies the entity used if the bone match is not found in the hierarchy - usually the entity the render component is attached to
+    /**
+     * Resolved skin bones to a hierarchy with the rootBone at its root.
+     *
+     * @param {import('../framework/entity.js').Entity} rootBone - A reference to the entity to be used as
+     * the root bone.
+     * @param {import('../framework/entity.js').Entity} entity - Specifies the entity used if the
+     * bone match is not found in the hierarchy - usually the entity the render component is attached to.
```
Comment (willeastcott) [inline src/scene/skin-instance.js:95]: `SkinInstance#resolve` does not currently appear as public API: https://api.playcanvas.com/classes/Engine.SkinInstance.html
Should this have an @ignore?

File: src/scene/skin-instance.js [hunk 2]
```diff
@@ -86,8 +86,14 @@ class SkinInstance {
         }
     }
 
-    // resolved skin bones to a hierarchy with the rootBone at its root.
-    // entity parameter specifies the entity used if the bone match is not found in the hierarchy - usually the entity the render component is attached to
+    /**
+     * Resolved skin bones to a hierarchy with the rootBone at its root.
```
Comment (willeastcott) [inline src/scene/skin-instance.js:90]:
```suggestion
     * Resolves skin bones to a hierarchy with the rootBone at its root.
```

File: src/scene/skin-instance.js (hunk 1 above)
Comment (kungfooman) [reply src/scene/skin-instance.js:95]: Thank you for pointing this out, making it public through the comment was unindented, I will add the `@ignore` tag (somehow I always forget it).

**Review by willeastcott — APPROVED** [review]
This is great - thank you!

**Pull request merged**